tests/sentry/snuba/test_discover.py

```diff
@@ -33,6 +33,66 @@ def setUp(self):
             project_id=self.project.id,
         )
 
+    def test_project_mapping(self):
+        other_project = self.create_project(organization=self.organization)
+        self.store_event(
+            data={"message": "hello", "timestamp": iso_format(before_now(minutes=1))},
+            project_id=other_project.id,
+        )
+
+        result = discover.query(
+            selected_columns=["project", "message"],
+            query="",
+            params={"project_id": [other_project.id]},
+            orderby="project",
+        )
+
+        data = result["data"]
+        assert len(data) == 1
+        assert data[0]["project"] == other_project.slug
```

Comment:
Should we ensure that other project fields are (not) present by asserting the keys in the response rows?

Reply:
Hmm the id fields are being stripped by the endpoint instead of at the query level, and we have a test for that already [here](https://github.com/getsentry/sentry/blob/master/tests/snuba/api/endpoints/test_organization_events_v2.py#L200)